package.json

```diff
@@ -24,51 +24,40 @@
     "build": "tsc",
     "build:types": "tsc --emitDeclarationOnly",
     "prepublishOnly": "npm run build",
-    "test": "jest --verbose",
```

Comment:
`test` command has been removed?

Reply:
It wasn't being used, so removed all references to tests

Reply:
There is still a whole testing framework in there... if we are removing remove it all.

Ideally we would have the test harness in there working even if its just a couple of basic tests.